test/libsolidity/ABIJson/function_type.sol

```diff
@@ -9,6 +9,7 @@ contract test {
 //     "inputs":
 //     [
 //       {
+//         "internalType": "function",
```

Comment:
Maybe we could add parameters and return types here.

Reply:
Yes, I think that could be useful (also to be able to look it up again).